ai_agents/agentic-form-filler/llm.py

```
@@ -0,0 +1,134 @@
+def get_client():
+    """Return the shared MiniMax client instance."""
+    return client
+
+def _parse_json_safely(text: str) -> dict:
+    """Extract and parse the first JSON object found in an LLM response string."""
+    match = re.search(r'\{.*\}', text, re.DOTALL)
+    if match:
+        try:
+            return json.loads(match.group(0))
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)